app/routes/app/ecosystem/metrics.tsx

```diff
@@ -0,0 +1,158 @@
+              title="Total Tokens Funded"
+              description="USD value of tokens funded across all challenges"
+              icon={<img alt="" src="/img/icons/dollar.svg" className="mr-2" />}
+            />
+            <StatsCard
+              figure={23000}
+              title="Total Tokens Claimed"
+              description="USD value of tokens claimed across all challenges"
+              icon={<img alt="" src="/img/icons/dollar.svg" className="mr-2" />}
+            />
+          </div>
+          <div className="bg-stone-200 rounded-md w-full h-44 border flex items-center">
+            <p className="py-20 mx-auto">Totally a chart</p>
+          </div>
+        </div>
+        <div className="space-y-5">
+          <p className="font-semibold text-md">Reputation</p>
+          <div className="grid grid-cols-1 md:grid-cols-3 gap-5 mx-auto items-center">
+            <StatsCard
+              figure={64000}
+              title="Total rMETRIC Issued"
+              description="Challenge winners and positive contributors earn rMETRIC"
+            />
+            <StatsCard
+              figure={23000}
+              title="Total rMETRIC Slashed"
+              description="Participants who don’t honor commitments on claimed challenges lose rMETRIC"
+            />
+            <StatsCard
+              figure="-50% every 90d"
+              title="Reputation Rate of Decay"
+              description="Ongoing decay ensures reputation scores represent recent contributions"
+            />
+          </div>
+          <div className="bg-stone-200 rounded-md w-full h-44 border flex items-center">
+            <p className="py-20 mx-auto">Totally a chart</p>
+          </div>
+        </div>
+      </main>
+      <aside className="md:w-1/4 lg:w-1/5 pt-11">
+        <SearchAndFilter />
+      </aside>
+    </section>
+  );
+}
+
+function StatsCard({
+  figure,
+  title,
+  description,
+  icon,
+}: {
+  figure: string | number;
+  title: string;
+  description?: string;
+  icon?: React.ReactNode;
+}) {
+  return (
+    <div className="flex flex-col gap-1 items-center border bg-blue-300/[.05] rounded-lg w-full h-full py-16 px-3">
+      <div className="flex">
+        {icon}
+        <p className="text-sky-500 text-3xl text-center">{figure}</p>
+      </div>
+      <p className="text-stone-500 text-lg text-center">{title}</p>
+      <p className="text-neutral-400 text-md text-center">{description}</p>
+    </div>
+  );
+}
+
+function SearchAndFilter() {
+  const submit = useSubmit();
+  const formRef = useRef<HTMLFormElement>(null);
+
+  const handleChange = () => {
+    if (formRef.current) {
+      submit(formRef.current, { replace: true });
+    }
+  };
+
+  return (
+    <ValidatedForm
+      formRef={formRef}
+      method="get"
+      noValidate
+      validator={validator}
+      onChange={handleChange}
+      className="space-y-3 p-4 border border-gray-300/50 rounded-lg bg-blue-300 bg-opacity-5 text-sm sticky top-0"
```

Comment:
minor thing: I _think_ you can move up the `sticky top-0` to the `aside` to make this component more agnostic to layout. SearchAndFilter then could get reused elsewhere (not that it will be).

Reply:
all good then